### PR TITLE
blunt replacement of setup.py -> use d2to1

### PR DESCRIPTION
**Pull request by feth**
simply replaced setup.py by the d2to1 boilerplate

### CODE REVIEW

**Comment by feth** [comment]
This now needs a correct setup.cfg file. Will do.

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,5 @@ bin/
 lib/
 local/
 include/
+venv_installed.stamp
```
Comment (Natim) [inline .gitignore]: I think it could be better to create the file here: var/stamps/venv_installed.

File: Makefile [hunk 2]
```diff
@@ -1,22 +1,36 @@
 PYTHON2 = `which python2 python2.7 python2.6 | head -1`
```
Comment (Natim) [inline Makefile:1]: :+1:

File: Makefile [hunk 3]
```diff
@@ -1,22 +1,36 @@
...
+DAYBED_EGG=lib/python2.7/site-packages/daybed.egg-link
```
Comment (Natim) [inline Makefile]: :-1: If it is python2.6 the path will be wrong.

`find lib/ -name "daybed.egg-link"`

**Comment by Natim** [comment]
Just two things but it looks good.

File: setup.cfg [hunk 4]
```diff
@@ -1,27 +1,29 @@
-[nosetests]
-match = ^test
-nocapture = 1
-cover-package = daybed
-with-coverage = 1
-cover-erase = 1
+[metadata]
+name = daybed
+version=0.0
```
Comment (almet) [inline setup.cfg]: should update to 0.1

File: setup.cfg [hunk 5]
```diff
@@ -1,27 +1,29 @@
...
+description=daybed
+description-file=
+    README.rst
+    CHANGES.rst
+classifiers=
+    Programming Language :: Python
+    PrFramework :: Pylons
```
Comment (almet) [inline setup.cfg]: "Pr" should be removed

File: setup.cfg [hunk 6]
```diff
@@ -1,27 +1,29 @@
...
+    PrTopic :: Internet :: WWW/HTTP
+    PrTopic :: Internet :: WWW/HTTP :: WSGI :: Application
```
Comment (almet) [inline setup.cfg]: ditto for these two

**Comment by Natim** [comment]
Thank you feth

**Comment by almet** [comment]
I second you on this, :+1:

**Comment by almet** [comment]
No, really.

**Comment by feth** [comment]
dimanche 2012-10-21 à 01:32:56 (CEST), Alexis Metaireau a écrit:

> I second you on this, :+1:

This would not have been possible were it not for alexis and natim.

**Comment by almet** [comment]
What?

**Pull request merged**